ui/src/app/components/resources/resource-list.spec.tsx

```diff
@@ -33,66 +35,41 @@ const COHORT: WorkspaceResource = {
 
 describe(ResourceList.name, () => {
   it('should render when there are no resources', async () => {
-    const wrapper = mount(
-      <MemoryRouter>
-        <ResourceList workspaceResources={[]} />
-      </MemoryRouter>
-    );
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.exists()).toBeTruthy();
+    renderWithRouter(<ResourceList workspaceResources={[]} />);
+    await screen.findByTestId('resources-table');
 
     // no resources are rendered
+    expect(screen.queryByText(/item type/i)).not.toBeInTheDocument();
     expect(
-      resourceTableColumns(wrapper).at(RESOURCE_TYPE_COLUMN_NUMBER).exists()
-    ).toBeFalsy();
-    expect(
-      resourceTableColumns(wrapper).at(NAME_COLUMN_NUMBER).exists()
-    ).toBeFalsy();
+      screen.queryByPlaceholderText(/search name/i)
+    ).not.toBeInTheDocument();
   });
 
   it('should render a cohort resource', async () => {
-    const wrapper = mount(
-      <MemoryRouter>
-        <ResourceList
-          workspaces={[workspaceDataStub]}
-          workspaceResources={[COHORT]}
-        />
-      </MemoryRouter>
+    renderWithRouter(
+      <ResourceList
+        workspaces={[workspaceDataStub]}
+        workspaceResources={[COHORT]}
+      />
     );
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.exists()).toBeTruthy();
 
-    expect(
-      resourceTableColumns(wrapper)
-        .at(RESOURCE_TYPE_COLUMN_NUMBER)
-        .find('div')
-        .first()
-        .text()
-    ).toBe('Cohort');
-    expect(
-      resourceTableColumns(wrapper)
-        .at(NAME_COLUMN_NUMBER)
-        .find('div')
-        .first()
-        .text()
-    ).toBe(COHORT_NAME);
+    await screen.findByTestId('resources-table');
```

Comment:
Not searching within columns, because RTL does not seem to offer a good way of doing that.
Since these tests only have a single row, I think these tests are sufficient.